webapplication/publisher/views.py

```diff
@@ -62,9 +62,10 @@ class ResultListAPIView(ListAPIView):
 
     def get_queryset(self):
         queryset = super().get_queryset()
+        request_id = self.request.query_params.get('request_id')
```

Comment:
This can be None, see example at https://www.django-rest-framework.org/api-guide/filtering/#filtering-against-query-parameters